src/resolvers/donationResolver.test.ts

```diff
@@ -783,7 +784,12 @@ function saveDonationTestCases() {
     });
     assert.isTrue(donation?.isTokenEligibleForGivback);
     assert.isOk(donation);
-    assert.isNull(donation?.userId);
+    const user = await User.findOne({
+      walletAddress: SEED_DATA.FIRST_USER.walletAddress,
+    });
+    // there will always be an userId present
+    // login resolvers always create an user even if profile not completed
```

Comment:
When we login with wallet it will always have a user id related, if not it will create it. (LoginWallet Mutation). LoginResolver.

I don't think I missed any instance the user is not created. But,
Correct me if I am wrong here @mohammadranjbarz , if you find any instance of user not created when logged in. I think it always goes through the loginWallet mutation.